src/zenml/integrations/deepchecks/__init__.py

```diff
@@ -0,0 +1,42 @@
+#  Copyright (c) ZenML GmbH 2021. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""
+The DeepChecks integration provides a way to validate your data in your pipelines.
```

Comment:
```suggestion
The Deepchecks integration provides a way to validate your data in your pipelines.
```

That's how they do it on their website.